esti/lakectl_local_test.go

```diff
@@ -406,3 +407,215 @@ func TestLakectlLocal_commit(t *testing.T) {
 		})
 	}
 }
+
+func TestLakectlLocal_interruptedCommit(t *testing.T) {
+	tmpDir := t.TempDir()
+	fd, err := os.CreateTemp(tmpDir, "")
```

Comment:
All tests share the same subdirectory.  So this test is brittle: If you add more cases and/or skip some cases it may break as the directory is not cleaned up between successive calls to `t.Run()`.  Best create a fresh temp directory inside each `t.Run()`.

Reply:
Are you sure about that? according to MkdirTemp function documentation- `MkdirTemp creates a new temporary directory in the directory dir and returns the pathname of the new directory. The new directory's name is generated by adding a random string to the end of pattern`